### PR TITLE
Return v2 and for to mapings

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: ops/mappings.json [hunk 1]
```diff
@@ -588,6 +714,48 @@
...
+        "dao": "package",
+        "mapping": "GenesisProtocol",
+        "arcVersion": "0.1.2-rc.2"
+      },
+      {
+        "name": "DAORegistry",
+        "contractName": "DAORegistryInstance",
+        "dao": "package",
+        "mapping": "DAORegistry",
+        "arcVersion": "0.1.2-rc.4"
```
Comment (orenyodfat) [inline ops/mappings.json:743]: this is rc.4 ?!

Comment (ben-kaufman) [reply ops/mappings.json:743]: I returned both, though I'm not sure we actually have any daos using 4, should I keep it anyway or remove it?

Comment (orenyodfat) [reply ops/mappings.json:743]: keep it anyhow

Comment (ben-kaufman) [reply ops/mappings.json:743]: ok, so please just approve :)

**Review by orenyodfat — APPROVED** [review]

**Pull request merged**